pkg/minishift/clusterup/clusterup.go

```diff
@@ -165,12 +164,13 @@ func EnsureHostDirectoriesExist(host *host.Host, dirs []string) error {
 }
 
 // DetermineOcVersion returns the oc version to use.
-// If the requested version is < the base line version we use baseline oc to provision the requested OpenShift version.
-// If the requested OpenShift version is >= the baseline, we align the oc version with the requested OpenShift version.
+// If the requested version is < v3.7.0 we will use oc binary v3.6.0 to provision the requested OpenShift version.
+// If the requested OpenShift version is >= v3.7.0, we align the oc version with the requested OpenShift version.
+// Check Minishift github issue #1417 for details.
 func DetermineOcVersion(requestedVersion string) string {
-	valid, _ := ValidateOpenshiftMinVersion(requestedVersion, version.GetOpenShiftVersion())
+	valid, _ := openshiftVersion.IsGTEBaseVersion(requestedVersion, constants.BackwardIncompatibleOcVersion)
```

Comment:
GTE => Greater Than or Equal? Too ambiguous...
Please, provide better naming

Reply:
@gbraad What about IsGreaterThanVersion or IsGreaterThanBaseVersion

Reply:
@LalatenduMohanty @gbraad , I would say `IsGreaterThanEqualToBaseVersion` as `IsGreaterThanBaseVersion` or `IsGreaterThanVersion` is incomplete and one could easily think that this function is skipping equally check.

Reply:
Can this be solved in a more explicit way? This feels confusing

Reply:
@LalatenduMohanty `IsGreaterOrEqualTo`